lib/promscrape/scrapework.go

```diff
@@ -354,7 +356,12 @@ func (sw *scrapeWork) run(stopCh <-chan struct{}, globalStopCh <-chan struct{})
 				// Too big jitter. Adjust timestamp
 				timestamp = t
 			}
+			missedTick := (time.Since(prevTick) / scrapeInterval) - 1
```

Comment:
Seems like the above code already did some missing ticker check
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/af5379933210fc09311ea74f234c44732d81c0a3/lib/promscrape/scrapework.go#L353-L356

Maybe we can reuse it like
```
			if d := math.Abs(float64(t - timestamp)); d > 0  {
				if d/float64(scrapeInterval.Milliseconds()) >= 1 {
					scrapesSkipped.Inc()
				}
				// Too big jitter. Adjust timestamp
				timestamp = t
			}
```

Reply:
Thanks! Please see updated